https://issues.redhat.com/browse/ACM-11010 2.10 (#6337)
open-cluster-management/rhacm-docs: governance/compliance_operator_policy.adoc

[#compliance-operator-policy]
= Compliance Operator policy

> The Compliance Operator assists users by automating the inspection of numerous technical implementations and compares those against certain aspects of industry standards, benchmarks, and baselines; the Compliance Operator is not  an auditor. In order to be compliant or certified within these various standards, you need to engage an authorized auditor such as a Qualified Security Assessor (QSA), Joint Authorization Board (JAB), or other industry recognized regulatory authority to assess your environment.  

Author: Suggestion after revised by Brandi

```suggestion
You can use the Compliance Operator to automate the inspection of numerous technical implementations and compare those against certain aspects of industry standards, benchmarks, and baselines. The Compliance Operator is not an auditor. To be compliant or certified with these various standards, you need to engage an authorized auditor such as a Qualified Security Assessor (QSA), Joint Authorization Board (JAB), or other industry recognized regulatory authority to assess your environment.  

Recommendations that are generated from the Compliance Operator are based on generally available information and practices regarding such standards, and might assist you with remediations, but actual compliance is your responsibility. Work with an authorized auditor to achieve compliance with a standard.

For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
```

> The Compliance Operator makes recommendations based on generally available information and practices regarding such standards and may assist with remediations, but actual compliance is your responsibility. You are required to work with an authorized auditor to achieve compliance with a standard.
> 
> For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].

Author: ```suggestion
```

Recommendations that are generated from the Compliance Operator are based on generally available information and practices regarding such standards, and might assist you with remediations, but actual compliance is your responsibility. Work with an authorized auditor to achieve compliance with a standard.

For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].

[#compliance-operator-policy-overview]
== Compliance Operator policy overview

You can install the Compliance Operator on your managed cluster by using the Compliance Operator policy. The Compliance operator policy is created as a Kubernetes configuration policy in {product-title-short}. {ocp-short} supports the compliance operator policy.

> //is this note valid? who can confirm this? Is it important to mention which architecture do not support the Compliance Operator?

Author: ```suggestion

//is this note valid? who can confirm this? Is it important to mention which architecture do not support the Compliance Operator?
```

[#compliance-operator-policy]
= Compliance Operator policy

You can use the Compliance Operator to automate the inspection of numerous technical implementations and compare those against certain aspects of industry standards, benchmarks, and baselines. The Compliance Operator is not an auditor. To be compliant or certified with these various standards, you need to engage an authorized auditor such as a Qualified Security Assessor (QSA), Joint Authorization Board (JAB), or other industry recognized regulatory authority to assess your environment.  

Recommendations that are generated from the Compliance Operator are based on generally available information and practices regarding such standards, and might assist you with remediations, but actual compliance is your responsibility. Work with an authorized auditor to achieve compliance with a standard.

> For the latest updates, see the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/security_and_compliance/compliance-operator#compliance-operator-release-notes[Compliance Operator release notes].
> 
> The Compliance Operator makes recommendations based on generally available information and practices regarding such standards and may assist with remediations, but actual compliance is your responsibility. You are required to work with an authorized auditor to achieve compliance with a standard.

Author: ```suggestion
```
